Command/winzouStateMachineDebugCommand.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace winzou\Bundle\StateMachineBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+
+class winzouStateMachineDebugCommand extends ContainerAwareCommand
+{
+    /**
+     * @var array
+     */
+    protected $config;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure()
+    {
+        $this->addArgument('key', InputArgument::REQUIRED, 'A state machine key');
+
+        $this->setName('debug:winzou:state-machine');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->config = $this->getContainer()->getParameter('sm.configs');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        if (null === $input->getArgument('key')) {
+            $helper = $this->getHelper('question');
+
+            $question = new ChoiceQuestion(
+                '<question>Which state machine would you like to know about?</question>',
+                array_keys($this->config),
```

Comment:
There is an ugly error when this array is empty, can you add a if to avoid instancing `ChoiceQuestion` with an empty choice?